modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,128 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage_{context}""]
+
+= Creating an `arm64` boot image using the Azure image gallery
+ 
+To configure your heterogeneous cluster, you must create an `arm64` boot image and add it to your Azure machine set. The following procedure describes how to manually generate an `arm64` boot image. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
+. Login to your Azure account: 
++
+[source,terminal]
+----
+$ az login
+----
+. Create a storage account and upload the `arm64` virtual hard disk (VHD) to your storage account. The {product-title} installation program creates a resource group, however, the boot image can also be uploaded to a custom named resource group: 
++
+[source,terminal]
+----
+$ az storage account create -n ${STORAGE_ACCOUNT_NAME} -g ${RESOURCE_GROUP} -l westus --sku Standard_LRS <1>
+----
++
+<1> The `westus` object is an example region. 
++
+. Create a storage container using the storage account you generated:
++
+[source,terminal]
++
+----
+$ az storage container create -n ${CONTAINER_NAME} --account-name ${STORAGE_ACCOUNT_NAME}
+----
+. You must use the {product-title} installation program JSON file to extract the URL and `arch64` VHD name:
+.. Extract the `URL` field and set it to `RHCOS_VHD_ORIGIN_URL` as the file name:
++
+[source,terminal]
+----
+$ RHCOS_VHD_ORIGIN_URL=$(./openshift-install coreos print-stream-json | jq -r '.architectures.aarch64."rhel-coreos-extensions"."azure-disk".url')
+----
+.. Extract the `aarch64` VHD name and set it to `BLOB_NAME` as the file name:
++
+[source,terminal]
+----
+$ BLOB_NAME=rhcos-$(./openshift-install coreos print-stream-json | jq -r '.architectures.aarch64."rhel-coreos-extensions"."azure-disk".release')-azure.aarch64.vhd
+----
+. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container: 
```

Comment:
Doc guidelines recommend including an explicit instruction to "run" or "enter" the command. E.g.
```suggestion
. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container by running the following command: 
```